src/client.rs

```diff
@@ -370,14 +355,14 @@ impl Client {
     }
 
     /// Construct shell command.
-    pub fn command_shell(command: String) -> mesos::CommandInfo {
+    pub fn command_shell(&self, command: String) -> mesos::CommandInfo {
```

Comment:
Why self? You are not using it, right?

Reply:
Just to unify between `client.command_shell` and `Client::command_shell`. Ideally we have proper builders for the Mesos Protobufs and the client would just construct calls. Should I do so in this diff?

Reply:
Not sure. But try to run `cargo +nightly clippy`. I am sure, it will complain about this.